### PR TITLE
Update readmes

### PR DESCRIPTION
**Pull request by Sukh-P**
# Pull Request

## Description

Small updates to the readme to reflect the current repo structure and content

### Fixes 

Out of date information

## How Has This Been Tested?

NA - only readme updates

## Checklist:

- [ ] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: ocf_datapipes/training/example/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ These are example datapipes using the `ocf_datapipes` library
 
 ## Simple PV
 
-`simple_pv.py` has a training pipeline for just loading PV data.
+[`simple_pv.py`](simple_pv.py) has a training pipeline for just loading PV data.
```
Comment (Sukh-P) [inline ocf_datapipes/training/example/README.md:7]: Removed one of the example diagrams and file mentioned that is no longer in the repo and then just reordered examples from simplest to most complex and added links, more clear to see that [here](https://github.com/openclimatefix/ocf_datapipes/blob/087d022c063eed28622544d649e8db98aa488d58/ocf_datapipes/training/example/README.md), just the order

**Review by peterdudfield — APPROVED** [review]
Thank @Sukh-P for putting this together

**Comment by peterdudfield** [comment]
@all-contributors please add @code for doc

**Comment by peterdudfield** [comment]
@all-contributors please add @Sukh-P  for doc

**Pull request merged**